core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -144,6 +144,12 @@ var ExplorationEditorMainTab = function() {
   var solicitAnswerDetailsCheckbox = element(
     by.css('.protractor-test-solicit-answer-details-checkbox'));
 
+  /*
+   * Symbols
+   */
+  var tickMark = element(
```

Comment:
Rename this to correctAnswerCheckbox (or something around that)

Reply:
Updated